compiler/noirc_frontend/src/hir/def_collector/dc_crate.rs

```diff
@@ -463,7 +464,7 @@ fn collect_trait_impl_methods(
 
             if overrides.len() > 1 {
                 let error = DefCollectorErrorKind::Duplicate {
-                    typ: DuplicateType::Function,
+                    typ: DuplicateType::TraitImplementation,
```

Comment:
Is it the trait implementation that is duplicated? I would have thought it is the function/method within the trait impl.

Reply:
it's outdated